ext/gd/gd.stub.php

```diff
@@ -96,30 +100,30 @@ function imagecreatefrombmp(string $filename): GdImage|false {}
 function imagecreatefromtga(string $filename): GdImage|false {}
 #endif
 
-function imagexbm(GdImage $im, ?string $filename, int $foreground = UNKNOWN): bool {}
+function imagexbm(GdImage $im, ?string $filename, ?int $foreground = null, ?int $filters = null): bool {}
```

Comment:
It seems to me that the forth parameter was missing from the stub, and not even the documentation mentions it. Or am I missing something?

Reply:
Indeed, that parameter is allowed, but it's actually unused. I shall have a word with the GD extension maintainer. ;)

Reply:
Can we just drop that argument?

Reply:
Haha :D Can we remove that parameter then? It's not even documented so that's why I think it might be possible. Or that's ok to uncover it in the stubs?

Reply:
Yes, the param should be dropped. It's about https://github.com/php/php-src/blob/baa285881fbdef24d5a737774d94bd76f422bf9f/ext/gd/gd.c#L4121, where the $basefilter parameter is only relevant for PNG images. From a quick look it seems the other image*() function don't list that parameter in the stubs.